docs/release-notes/cloud/cloud-2023-03.md

```diff
@@ -0,0 +1,88 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud March 2023.
+included_jira_versions: ['2023.03']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/6.7.116
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/7.0.9
+---
+
+# Cloud March 2023
+
+These release notes are for the Codacy Cloud updates during March 2023.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   Added support for the following plugins:
+    -   [<span class="skip-vale">@tailwindcss/line-clamp</span>](https://www.npmjs.com/package/@tailwindcss/line-clamp) for ESLint (TS-288)
+    -   [<span class="skip-vale">pylint-beam</span>](https://github.com/kvudata/pylint-beam) for Pylint (Python 3) (TS-274)
+    -   [<span class="skip-vale">PHPCompatibilityWP</span>](https://github.com/PHPCompatibility/PHPCompatibilityWP) for PHP_CodeSniffer (TS-56)
+-   Some Enterprise plans now allow downloading a CSV list of all organization members from the **People** page. This feature is also available on all plans using the API endpoint [listPeopleFromOrganizationCsv](https://api.codacy.com/api/api-docs#listpeoplefromorganizationcsv). (PLUTO-388)
+-   Codacy now supports up to 10 coding standards per organization, allowing you to [optimize and apply quality settings to multiple repositories <span class="skip-vale">quickly</span>](../../organizations/using-a-coding-standard.md). For more details, [see the announcement on our blog](https://blog.codacy.com/organization-coding-standards-just-got-better/). (IO-358)
+
+    ![Multiple coding standards](../images/io-358.png)
+
+## Bug fixes
+
+-   Updated the Codacy Analysis CLI GitHub Action so it's no longer rate-limited by the GitHub API. (TS-109)
+-   Fix permission issue when a user has minimal access on GitLab. (PLUTO-468)<!--TODO Clarify -->
```

Comment:
Hi @lolgab, could you help clarify in what scenario the issue [PLUTO-468](https://codacy.atlassian.net/browse/PLUTO-468) happened to customers?

Reply:
@angelabarros tagging you instead since Lorenzo is on vacation and you [approved his pull request](https://github.com/codacy/remote-provider-service/pull/210#pullrequestreview-1358859184). :wink:

Reply:
I believe this was part of the problem with the Repo's name not syncing properly in Gitlab. This was discovered by Lorenzo and was tackled as well because it was an easy fix. The NullPointerExceptions were being thrown before when being converted and now with this fix it resolved this issue.

Reply:
Thanks for clarifying, @angelabarros! In this case I'll remove the release note since it doesn't look like we had actual users report this issue.

Reply:
My general opinion about this is that even when an issue is discovered by us and not by users, there is value in documenting the fix so users that were affected and noticed a change in behavior can know it was because of a bug fix. Not all users having issues are reporting them, I don't report misbehaviors in most of the software I use, but I tend to read changelogs.
Anyway, thank you @prcr and @angelabarros, thank you for handling this while I was on vacation ❤️